File: Theory and Practice of Computation/Prac_1/blab.pl
get_vars :-
    write('message?'),
    nl,
    read(X),
    write('repeats?'),
    read(Y),
    blab_print(Y,X).

blab_print(N,X) :-
    N>0,
    M is N-1,
    write(X),
    nl,
    blab_print(M,X).

File: Theory and Practice of Computation/Prac_1/blab.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int repeats;
    char message[40];

    printf("Message?");
    scanf("%[^\n]%*c", &message);
    printf("repeats?");
    scanf("%d", &repeats);


    for(int i=0;i<repeats;i++){
        printf("%s\n", message);
    }

    return 0;
}